components/locale/ku_IQ.tsx

```diff
@@ -1,9 +1,13 @@
 import Pagination from 'rc-pagination/lib/locale/ku_IQ';
-import DatePicker from '../date-picker/locale/ku_IQ';
-import TimePicker from '../time-picker/locale/ku_IQ';
-import Calendar from '../calendar/locale/ku_IQ';
+import DatePicker from '../date-picker/locale/kmr_IQ';
+import TimePicker from '../time-picker/locale/kmr_IQ';
+import Calendar from '../calendar/locale/kmr_IQ';
 import { Locale } from '../locale-provider';
 
+// please use antd/es/locale/kmr_IQ instead
+// keep this file for compatibility
+// https://github.com/ant-design/ant-design/issues/25778
+
```

Comment:
Keep `antd/es/locale/ku_IQ` for compatibility.

Reply:
What I understand is to add a new locale named `kmr_IQ` and keep it consistent with `ku_IQ` instead of renaming it.

Reply:
It is what it is.

Reply:
How about #26035?

Reply:
Removed all `ku_IQ` but `components/locale/ku_IQ.tsx`, please check again.